home.md

```diff
@@ -0,0 +1,75 @@
+<img width="500" align="right" alt="illustration" src="https://user-images.githubusercontent.com/82679007/150655220-d023b26e-ceab-4b37-9d66-0ad6b20f0628.png">
+
+# Mentorship...for you!
+
+### Do you need guidance on a project? Need help learning how to contribute to open source? Guidance in settling into the Galaxy ecosystem? More?
+
+## About Galaxy Mentorship Network Program
+Galaxy Mentorship Network program is an initiative by the Galaxy Community to build her members in open source projects. 
+
+It is run on an open enrollment basis where mentees and mentors can apply at any time of need, this is so especially because we have a lot of new members who join regularly and may immediately need someone to guide them. This is a program by Galaxy members and for Galaxy members.
+
+## Vision
+ We hope to increase the proficiency of members in the use of Galaxy tools and servers, contributing to open source projects as well as becoming active members of any of our channels and working groups.
+
+### Benefits of the GMN program
+
+-Mentees receive expert advice on their projects, goals, and the community.
```

Comment:
```suggestion
- Mentees receive expert advice on their projects, goals, and the community.
```

Reply:
Same for all lines that are list. You need also to remove the empty line between list element